src/services/identity/user/user-management/modules/user-management-modal/UserUpdateModal.vue

```diff
@@ -232,15 +233,15 @@ export default {
                 if (!(validationState.isPasswordValid && validationState.isPasswordCheckValid)) return;
             }
             const data = {
-                user_id: formState.user_id,
-                name: formState.name,
-                email: formState.email,
+                user_id: state.user_id,
```

Comment:
```suggestion
                user_id: state.userId,
```
변수명 userId 로 바꾸는게 좋을 것 같아요~